### DISCUSSION TITLE
How can you customize record title on attach action

### DISCUSSION THREAD

**Discussion by Grolicek** [filamentphp/filament#13096] [Help] [answered]
### Package

Table builder

### Package Version

v3.2.83

### How can we help you?

Hello,

I would like to know if it's possible to customize record title more than just a different attribute. I would like to combine multiple attributes in to one title. Is it possible?

![image](https://github.com/filamentphp/filament/assets/58747066/c9d56a8f-4ea4-4512-9f2f-519f10f84006)

Thanks for any help...

**Comment by petrisorcraciun** [comment]
if I understood well, you can use https://filamentphp.com/docs/3.x/forms/fields/select#returning-custom-search-results

**Reply by Grolicek** [reply]
I'm already doing this, isn't this supposed to the trick?

This just shows for each record title a lowercase name of model from which it is...
![image](https://github.com/filamentphp/filament/assets/58747066/b5f2d525-a7d1-4cb9-adcc-3676b70c0b05)

```php
->headerActions([
    Tables\Actions\AttachAction::make()
        ->form(fn(AttachAction $action): array => [
            $action->getRecordSelect()
                ->getOptionLabelFromRecordUsing(fn (Model $record) => "{$record->id} {$record->start}")
        ]),
])
```

**Comment by leandrocfe** [comment, marked as answer]
```php
Tables\Actions\AttachAction::make()
    ->recordTitle(fn (Model $record): string => "{$record->id} {$record->start}")
```

**Reply by Grolicek** [reply]
Thanks, solved my problem.
